docs/source/json-api/index.rst

```diff
@@ -316,7 +314,7 @@ Examples
 Create a new Contract
 *********************
 
-See the request documentation below on how to create an instance of ``Iou`` contract from the :doc:`Quickstart guide </app-dev/bindings-java/quickstart>`:
+See the request documentation below on how to create an instance of ``Iou`` contract from the :doc:`Getting Started guide </app-dev/bindings-java/quickstart>`:
```

Comment:
@gerolf-da do we want to keep it as Quickstart guide for now?
Carol mentioned that `we are replacing Quickstart with Getting Started in SDK 1.0`...
changing the title to Getting Started, links does not change.

Reply:
Please keep the title as quickstart. We should continue to use unambiguous names between the two. Ideally the JSON API docs use templates and examples from the new getting started guide.

Reply:
reverted this part back to Quickstart guide

Reply:
Shall we call it the "Java bindings quickstart guide" to be even more clear?